### ISSUE TITLE
reproduce result

### ISSUE THREAD

**Issue by zhujun3753** [alexrich021/3dvnet#7]
Thank you for your great work!
In `mv3d/lightningmodel.py`, there is no loss for updating weights of ` self.refine_quarter,  self.refine_half,  self.refine_full` .
```python
   # starting size --> 1/4
depth_pred = F.interpolate(depth_pred.unsqueeze(1), feats_quarter.shape[-2:], mode='nearest').squeeze(1)
depth_pred = self.refine_quarter(feats_quarter[ref_idx], depth_pred.unsqueeze(1)).squeeze(1)
# 1/4 supervision
depth_gt_quarter = F.interpolate(batch.depth_images.unsqueeze(1), depth_pred.shape[-2:], mode='nearest').squeeze(1)
loss_refine_quarter = self.mae_loss(depth_pred, depth_gt_quarter, self.hparams.depth_test['depth_interval'])
metrics = calc_2d_depth_metrics(depth_pred, depth_gt_quarter)
metrics['loss_2d'] = loss_refine_quarter
out['quarter'] = metrics
```
Could you please release the compelte code for training the pretrianed model ?
Thanks.

**Comment by alexrich021** [comment]
Hey, thanks for your interest!

Great catch. I was experimenting with a surface normal loss prior to uploading the code and must have accidentally removed the refinement loss when I removed the surface normal loss.

Basically you just need to add `loss += loss_refine_*` after each refinement step. I'll push an update soon that will fix this.

Let me know if you have any other issues
-Alex

**Comment by alexrich021** [comment]
Just pushed the fix

-Alex

**Comment by zhujun3753** [comment]
Thanks a lot!!!
